packages/server/graphql/types/DragEstimatingTaskPayload.ts

```diff
@@ -21,13 +23,7 @@ export const DragEstimatingTaskSuccess = new GraphQLObjectType<any, GQLContext>(
     },
     stage: {
       type: GraphQLNonNull(EstimateStage),
-      resolve: async ({meetingId, stageId}, _args, {dataLoader}) => {
-        const meeting = await dataLoader.get('newMeetings').load(meetingId)
-        const {phases} = meeting
-        const estimatePhase = phases.find((phase) => phase.phaseType === 'ESTIMATE')
-        const {stages} = estimatePhase
-        return stages.find((stage) => stage.id === stageId)
-      }
+      resovle: resolveStage(NewMeetingPhaseTypeEnum.ESTIMATE)
```

Comment:
Just noticed a tiny typo here

Reply:
nice catch! i've been thinking about adding typings to the newMutation generator. i'll get that in